doc/source/reference/series.rst

```diff
@@ -327,6 +328,7 @@ Datetime properties
    Series.dt.tz
    Series.dt.freq
    Series.dt.unit
+   Series.dt.normalize
```

Comment:
@mroeschke - this and the one above were added in #52503. They were missing from the API docs. Does this placement look good?

Reply:
Looks good, thanks!